writing-visualising/dashboards.qmd

```diff
@@ -502,8 +502,8 @@ Each test should produce one "expected results" JSON file. You can track particu
 For example, the [SCAP LA school places scorecards app](https://github.com/dfe-analytical-services/la-school-places-scorecards/blob/main/tests/shinytest/UI_tests.R) has tests that check that the app functions as expected when different combinations of filters are selected.
 
 Each test is separated out by comments, clearly stating what is being tested.
- - **app$setInputs()** tells the UI test to select an input in the app
- - **app$snapshot()** tells the UI test which outputs to capture (i.e. which outputs you'd expect to change and want to test)
+ - `app$setInputs()` tells the UI test to select an input in the app
```

Comment:
These bullet points aren't showing as expected, I think you might need an extra blank line before they start? [this branch]:
![image](https://github.com/dfe-analytical-services/analysts-guide/assets/52536248/9c7b12ad-894d-4652-8f08-23377341bd5b)

Though the main branch is even worse, so you've not broken it here [main branch that's live]:

![image](https://github.com/dfe-analytical-services/analysts-guide/assets/52536248/da3c74c0-a457-457e-80ea-8df40f07f710)